Userland/Applications/ImageViewer/main.cpp

```diff
@@ -65,6 +109,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto main_toolbar = TRY(toolbar_container->try_add<GUI::Toolbar>());
 
     auto widget = TRY(root_widget->try_add<ViewWidget>());
+    if (fallback_to_decode_binary_pixel_bitmap) {
+        widget->set_forced_bitmap_format(raw_bitmap_format.value());
+    }
+    if (force_dimensions) {
+        widget->set_forced_dimensions({ raw_width.value(), raw_height.value() });
+    }
```

Comment:
These one-line `if`-blocks do not need their braces (maybe remove them for `if (path)` below as well?)